calico/maintenance/ebpf/enabling-bpf.md

```diff
@@ -162,35 +162,7 @@ This section explains how to make sure your cluster is suitable for eBPF mode.
 
 In eBPF mode, {{site.prodname}} implements Kubernetes service networking directly (rather than relying on `kube-proxy`).  This means that, like `kube-proxy`,  {{site.prodname}} must connect _directly_ to the Kubernetes API server rather than via the API server's ClusterIP.
 
-First, make a note of the address of the API server:
-
-* If you have a single API server with a static IP address, you can use its IP address and port.  The IP can be found by running:
-
-  ```
-  kubectl get endpoints kubernetes -o wide
-  ```
-
-  The output should look like the following, with a single IP address and port under "ENDPOINTS":
-
-  ```
-  NAME         ENDPOINTS             AGE
-  kubernetes   172.16.101.157:6443   40m
-  ```
-
-  If there are multiple entries under "ENDPOINTS" then your cluster must have more than one API server.  In that case, you should try to determine the load balancing approach used by your cluster and use the appropriate option below.
```

Comment:
If there are multiple entries under "ENDPOINTS", then your cluster...

In this case, use the appropriate load balancing option below for your cluster.